DESCRIPTION

```diff
@@ -18,28 +18,47 @@ Depends:
     R (>= 3.6)
 Imports:
     bslib,
-    checkmate,
-    ggplot2,
+    checkmate (>= 2.1.0),
+    ggplot2 (>= 3.4.0),
     graphics,
     grDevices,
-    htmltools,
-    lifecycle,
+    htmltools (>= 0.5.4),
+    lifecycle (>= 0.2.0),
     methods,
     rtables (>= 0.6.0.9004),
-    shiny,
+    shiny (>= 1.6.0),
     shinyjs,
-    shinyWidgets,
-    styler
+    shinyWidgets (>= 0.5.1),
+    styler (>= 1.2.0)
 Suggests:
-    knitr,
-    lattice,
-    magrittr,
+    knitr (>= 1.42),
+    lattice (>= 0.18-4),
+    magrittr (>= 1.5),
     png,
```

Comment:
When running `max_deps_check_result <- verdepcheck::max_deps_check("teal.widgets")` it gives this error:
```r
✔ Installed digest 0.6.33  (284ms)
✔ Installed ellipsis 0.3.2  (145ms)
✔ Installed evaluate 0.21  (161ms)
✔ Installed fansi 1.0.4  (179ms)
✔ Installed farver 2.1.1  (170ms)
✔ Installed fastmap 1.1.1  (151ms)
✖ Failed to build png 0.1-8
! Failed to build source package 'png', stdout + stderr (last 10 lines):

OE> *** installing help indices
OE> ** building package indices
OE> ** testing if installed package can be loaded
OE> Error: package or namespace load failed for ‘png’ in dyn.load(file, DLLpath = DLLpath, ...):
OE>  unable to load shared object '/private/var/folders/gy/xtc2dqrx62s36gj1p8l_9zxh0000gn/T/RtmpovRdqC/pkg-lib15bf14bca4328/png/libs/png.so':
OE>   dlopen(/private/var/folders/gy/xtc2dqrx62s36gj1p8l_9zxh0000gn/T/RtmpovRdqC/pkg-lib15bf14bca4328/png/libs/png.so, 0x0006): symbol not found in flat namespace '_png_create_info_struct'
OE> Error: loading failed
OE> Execution halted
OE> ERROR: loading failed
OE> * removing ‘/private/var/folders/gy/xtc2dqrx62s36gj1p8l_9zxh0000gn/T/RtmpovRdqC/pkg-lib15bf14bca4328/png’
Error in "stop_task_build(state, worker)" :
  ! Failed to build source package 'png'
```

Reply:
Both of them seem like a problem with a png library not present on your system.

Can you install `png` on your normal R session? (both the latest and v0.1-8)

```R
renv::install("png")
```

Reply:
I can and it's already installed.
```r
> renv::install("png")
The following package(s) will be installed:
- png [0.1-8]
These packages will be installed into "/Library/Frameworks/R.framework/Versions/4.3-x86_64/Resources/library".

Do you want to proceed? [Y/n]: y

# Installing packages --------------------------------------------------------
- Installing png ...                            OK [copied from cache]
Successfully installed 1 package in 55 milliseconds.
```

Reply:
I've asked the group. Let's see if anyone else has the issue. Otherwise, we can confirm that my laptop is trying to sabotage my work.

Reply:
Let me try to do this with a different version of R